src/main/webapp/WEB-INF/user.jsp

```diff
@@ -51,7 +51,9 @@
     %>
           <div class="message-div">
             <div class="message-header">
-              <%= messages.get(i).getUser() %> - <%= new Date(messages.get(i).getTimestamp()) %>
+              User: <%= messages.get(i).getUser() %> -
+              Time: <%= new Date(messages.get(i).getTimestamp()) %> -
+              Sentiment Score: <%= messages.get(i).getSentimentScore() %>
```

Comment:
I like the score descriptor, not sure if you wanted to add it to the feed messages as well or not, but if not, LGTM!

Reply:
I'll go ahead and add that to the feed, too. Thanks!

Reply:
Actually, that's still using HTML so I might need to add some extra logic to the servlet. So, I think I will merge without it, and update that in a separate PR after we discuss if we want the feed to be SSR or not